python/documentation/create_operations_doc.py

```diff
@@ -0,0 +1,71 @@
+''' Script for generating Python documentation from the ASCIIDOC documentation.
+
+
+
```

Comment:
Maybe one or two empty lines would be enough. 😄

Reply:
Fixed.